lib/bot.js

```diff
@@ -163,8 +163,13 @@ class Bot {
         return;
       }
 
+      const withMentions = text.replace(/@[^\s]+\b/g, m => {
+        const user = this.discord.users.get('username', m.substring(1));
+        return user ? user.mention() : 0;
```

Comment:
Shouldn't this be `return user ? user.mention () : m`? I.e. not `0`. Also, could probably rename m to `match` or something more readable.